tests/test_cputop.py

```diff
@@ -44,4 +44,4 @@ def test_cputop(self):
         expected = self.get_expected_output('cputop.txt')
         result = self.get_cmd_output('lttng-cputop')
 
-        self.assertMultiLineEqual(result, expected)
+        self.diff("cputop", result, expected)
```

Comment:
Single quotes again, and the same comment for all calls to `self.diff` in the other files.